relay-general/src/store/transactions.rs

```diff
@@ -395,16 +392,26 @@ mod tests {
             ..Default::default()
         });
 
-        assert_eq_dbg!(
-            process_value(
-                &mut event,
-                &mut TransactionsProcessor::new(None),
-                ProcessingState::root()
-            ),
-            Err(ProcessingAction::InvalidTransaction(
-                "trace context is missing op"
-            ))
-        );
+        process_value(
+            &mut event,
+            &mut TransactionsProcessor::new(None),
+            ProcessingState::root(),
+        )
+        .unwrap();
+        assert!(event.value().is_some());
+        // assert_eq!(
+        //     event
+        //         .value()
+        //         .unwrap()
+        //         .contexts
+        //         .value()
+        //         .unwrap()
+        //         .get("trace")
+        //         .unwrap()
+        //         .value()
+        //         .unwrap(),
+        //     "op"
+        // );
```

Comment:
I need help with this madness :)

@jan-auer or @untitaker, how can I check that the `op` was defaulted?

Reply:
Do this instead: https://github.com/getsentry/relay/blob/ab4cfdc70f53b0fd17f554af9a74378859d68630/relay-general/src/store/normalize/user_agent.rs#L180

Much simpler. You can write `@""` as string initially and then use [insta](https://crates.io/crates/insta) the cli-tool to approve the snapshot.

Reply:
Thanks!